test/test02/SBE16plus_01606330_2016_03_30.cnv

```diff
@@ -0,0 +1,42 @@
+* Sea-Bird SBE16plus  Data File:
+* FileName = C:\Users\jan079\Desktop\SBE16plus_01606330_2016_03_30.hex
+* Software version 2.4.1
+
+# name 0 = timeJV2: Time, Instrument [julian days]
+# name 1 = timeK: Time, Instrument [seconds]
+# name 2 = tv290C: Temperature [ITS-90, deg C]
+# name 3 = c0S/m: Conductivity [S/m]
+# name 4 = prdM: Pressure, Strain Gauge [db]
+# name 5 = sal00: Salinity, Practical [PSU]
+# name 6 = depSM: Depth [salt water, m], lat = -46.00
+# name 7 = sbeox0ML/L: Oxygen, SBE 43 [ml/l]
+# name 8 = sbeox0PS: Oxygen, SBE 43 [% saturation]
+# name 9 = sbeox0Mm/L: Oxygen, SBE 43 [umol/l]
+# name 10 = sbeox0Mm/Kg: Oxygen, SBE 43 [umol/kg]
+# name 11 = oxsolMm/Kg: Oxygen Saturation, Garcia & Gordon [umol/kg]
+# name 12 = v0: Voltage 0
+# name 13 = v1: Voltage 1
+# name 14 = v2: Voltage 2
+# name 15 = v3: Voltage 3
+# name 16 = v4: Voltage 4
+# name 17 = v5: Voltage 5
+# name 18 = potemp090C: Potential Temperature [ITS-90, deg C]
+# name 19 = sigma-�00: Density [sigma-theta, kg/m^3]
+# name 20 = flag:  0.000e+00
+
```

Comment:
This is all that I thought that is needed to parse the file, this part of the file is the generated output from the sea-bird DataConversion utility, the rest of the header is from the instrument. 
I had not parsed the file with the toolbox in the test, but included it only for reference, are you planning on parsing this file?